templates/solutions/ai/index.html

```diff
@@ -0,0 +1,571 @@
+      <hr />
+      <p>
+        <a href="https://ubuntu.com/engage/run-ai-at-scale">Download the whitepaper on NVIDIA DGX and Charmed Kubeflow&nbsp;&rsaquo;</a>
+      </p>
+      {{ 
+        image (
+        url="https://assets.ubuntu.com/v1/ba3a0335-Canonicla+nvidia.png",
+        alt="",
+        width="536",
+        height="66",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>AI at the edge</h2>
+    </div>
+    <div class="col">
+      <p>Drive real-time insight in the field and on the factory floor by running your AI models directly on edge devices.</p>
+      <p>Deploy compute where you need it to power AI use cases at the edge. Solve security, management and update challenges for your fleet with an OS purpose-built for IoT devices and optimised for AI.</p>
+      <hr />
+      <p>
+        <a href="/blog/edge-ai">Edge AI explained&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Professional services<br /> from open source experts</h2>
+    </div>
+    <div class="col">
+      <p>Shortage of expertise is among the most significant barriers to AI adoption. Bridge the skills gap with Canonical's professional services. Let our experts guide you through your optimal AI journey, or completely offload the operational burden of your AI stack with our managed service.</p>
+      <hr />
+      <a class="p-button" href="https://assets.ubuntu.com/v1/429359fe-MLOps%20advisory%20DS%2029.11.22.pdf">Read the datasheet</a>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="u-fixed-width">
+    <h2 class="p-text--small-caps">What customers say</h2>
+  </div>
+  <div class="row">
+    <hr class="p-rule"/>
+    <div class="col-3 col-medium-2 p-image-wrapper p-section">
+      {{
+        image (
+        url="https://assets.ubuntu.com/v1/dc69e636-firmus-black.png",
+        alt="Firmus",
+        width="100",
+        height="23",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+    <div class="col-6 col-medium-4">
+      <blockquote class="p-pull-quote u-no-margin u-no-padding">
+        <div class="p-section--shallow">
+          <p class="p-heading--4"><i>"We needed a cloud solution that was stable, reliable and performant. Canonical allowed us to do this by helping to design and deploy our cloud &mdash; and they helped us do this quickly."</i></p>
+        </div>
+        <hr class="p-rule"/>
+        <cite class="p-section"><span class="p-heading--5">Peter Blain</span><span style="float: right;">Director, Product and AI Firmus</span></cite>
```

Comment:
Why not using `u-float-right` util?

Reply:
This one may be as old as Vanilla itself 🤣 

But I take it as a sign that it's rarely needed 😉